kie-pmml-dependencies/pom.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kie-parent</artifactId>
+    <groupId>org.kie</groupId>
+    <version>7.40.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kie-pmml-dependencies</artifactId>
```

Comment:
I would suggest to use "kie-pmml-trusty-dependencies" to distinguish from old PMML implementation. I guess this module is mainly for end user. Am I correct?

Reply:
@jiripetrlik @danielezonca 
I've used "kie-pmml-dependency" for consistence with "kie-pmml-bom" (both are related to kie-pmml-trusty).
I would like to have the same naming convention, so
1) `kie-pmml-trusty-bom` /  `kie-pmml-trusty-dependencies`
                OR
2) `kie-pmml-bom` /  `kie-pmml-dependencies`
Pls let me know ASAP so that I can fix it - thk :)

Reply:
I would vote for the option one, because it will cause less confusion and users immediately know which implementation is used. I also like word "trusty" in the new implementation name. But this is only my subjective opinion and I will not complain if you decide for the option 2.

Reply:
`trusty` suffix is only a workaround for `kie-pmml-new` renaming, I would like not to rename **all** the modules just for that. I vote for option 2 :)

Reply:
@jiripetrlik @danielezonca 
So kie-pmml-bom / kie-pmml-dependencies
Thanks!